### DISCUSSION TITLE
Cache issue? Force to forced reload (ctrl+F5) on every page to see changes

### DISCUSSION THREAD

**Discussion by FabioEight** [firefly-iii/firefly-iii#6690] [General]
Hello,

I have a Firefly III instance on a RPi server and I'm reaching it locally.
I use Edge browser and when I add or change a transaction or any other data on Firefly, I have to force reload the page to see the changes.
This is unconfortable as I move through the sections of Firefly and see the old data. Then I remind to force reload and everything is good.
Anybody experienced similar issues? I had not the chance to test a different browser as I'm using Edge for all my personal stuff.

Thanks!

**Comment by JC5** [comment]
Is it running through Docker or are you hosting your own server?

**Comment by FabioEight** [comment]
It’s on docker!

**Reply by JC5** [reply]
Alright. Firefly III doesn't block your browser from caching the page, but it doesn't enforce a strong cache rule either. What's your setup like? Any config in the reverse proxy?

**Comment by FabioEight** [comment]
I’m using Cloudflare Tunnel for accessing Firefly III. Yet, I experience this also accessing the host locally.
I’m using Edge on latest version.
________________________________
Da: James Cole ***@***.***>
Inviato: Friday, December 9, 2022 8:57:12 PM
A: firefly-iii/firefly-iii ***@***.***>
Cc: Fabio Eight ***@***.***>; Author ***@***.***>
Oggetto: Re: [firefly-iii/firefly-iii] Cache issue? Force to forced reload (ctrl+F5) on every page to see changes (Discussion #6690)


Alright. Firefly III doesn't block your browser from caching the page, but it doesn't enforce a strong cache rule either. What's your setup like? Any config in the reverse proxy?

**Reply by JC5** [reply]
Sorry, I don't run Windows so this isn't something I can verify. But Firefly III does not force caching (nor will it block caching). I'm a little out of my depth here, sorry.
